rulebook.tex

```diff
@@ -1630,6 +1640,10 @@ \subsection{Task Fulfillment and Scoring}
 awarded. Performing a step for a later order which has not yet been
 announced cannot be awarded.
 
+For a competitive order, the same scoring scheme for in-time, delayed, and late
+delivery applies for the first delivery, with an addition of $10$ points for a
```

Comment:
This explicit score should be removed here, point to the table, reduce redundancies especially for the crucial scoring bit (we ran into ambiguities in the past if only one place was updated and the other forgotten).

Reply:
Makes sense. I did not change the sentence about giving no points to the second delivery, as this will most probably not change, and it helps to understand competitive orders.